src/vstart.sh

```diff
@@ -1019,7 +1019,8 @@ do_rgw()
     i=0
     for rgw in j k l m n o p q r s t u v; do
         echo start rgw on http${CEPH_RGW_HTTPS}://localhost:$((CEPH_RGW_PORT_NUM + i))
-        run 'rgw' $RGWSUDO $CEPH_BIN/radosgw -c $conf_fn --log-file=${CEPH_OUT_DIR}/rgw.$rgw.log ${RGWDEBUG} --debug-ms=1 -n client.rgw "--rgw_frontends=${rgw_frontend} port=$((CEPH_RGW_PORT_NUM + i))${CEPH_RGW_HTTPS}"
+        current_port=$((CEPH_RGW_PORT + i))
```

Comment:
why not use `$((CEPH_RGW_PORT_NUM + i))` here? as `$CEPH_RGW_PORT_NUM` is always a *number*, while `$CEPH_RGW_PORT_NUM` could have a trailing "s".